Onion_Dips/traffic_light/main.css

```diff
@@ -0,0 +1,49 @@
+
+
+body {
+  width: 600px;
+  height: 600px;
+}
+
+ul.command {
+  display: inline-block; 
+  margin-top: 100px;
+  float: left;
+}
+
+ul.command li {
+  border-radius: 10px;
+  margin-bottom: 90px;
+  height: 75px;
+  width: 115px;
+  background-color: grey;
+  color: white;
+  text-align: center;
+  font-size: 24pt;
+  font-family: Arial;
+  line-height: 75px;
+  font-weight: bold; 
+}
+
+ul.case {
+  display: inline-block;
+  margin-left: 40px;
+  height: 540px;
+  width: 200px;
+  background-color: #474747;
+  padding-top: 10px;
+  border-radius: 30px;
+    
+}
+
+ul.case li {
+  list-style-type: none;
```

Comment:
hi Molly, the bullet points are still displaying in the buttons, as you can see from this preview of your code: http://htmlpreview.github.io/?https://github.com/mmlincoln/website-l10n_homework/blob/master/Onion_Dips/traffic_light/traffic_light.html

I think it's because you changed the list-style only for the .case ul.
You still have time if you would like to change it and re-submit the assignment.